pages/index.md.erb

```diff
@@ -0,0 +1,64 @@
+# Landing pages
```

Comment:
Is this file required?

Reply:
Oh good catch, that is where I started mocking out the content, will delete.